src/components/ContextMenu/PrefModal.js

```diff
@@ -21,7 +21,6 @@ import "./PrefModal.css";
 
 const DEFAULT_PREFS = {
   // general
-  //dbcsDetect    : false,
```

Comment:
I think we should keep this, even there's no way to change this. It's kind of a general feature for BBS.

Reply:
I just rebased. So did you mean just removing this commit from the history?
https://github.com/robertabcd/PttChrome/pull/32/commits/f19dd642d1cad34ffe02571526d96e17e429c157

Reply:
Yeah, this would do.

Reply:
Done.